### ISSUE TITLE
Always the detected object is "person"

### ISSUE THREAD

**Issue by MayooranT** [LocalJoost/YoloHolo#8]
Hi, First of all, awesome work. This had been a starting point into one of my researches.
The issue is, an object is detected as person all the time. I skimmed through the codes, where I found it is getting the object which has the highest probability. But why is it always displaying "person"?
P.S. I am using the main branch. I tried the airplane too but the model translator had only airplane. May be not relevant to me.

Thanks in advance!

**Comment by MayooranT** [comment]
Lately, I found out it detects just fine once I deploy it on the HL2. Why is it not good on the remoting simulator in Unity?

**Comment by LocalJoost** [comment]
> Lately, I found out it detects just fine once I deploy it on the HL2. Why is it not good on the remoting simulator in Unity?

I am not sure. I never tried it via remoting. You might check if the app doesn't use your computer's webcam when using remoting - and then it probably recognizes you, in front of it :)

**Comment by MayooranT** [comment]
Good catch! also I learnt it later that feature is not available. https://learn.microsoft.com/en-us/answers/questions/710867/how-to-access-hl2-pv-camera-by-holographic-remotin
